web-services/common-util/src/main/java/datawave/resteasy/interceptor/CreateQuerySessionIDFilter.java

```diff
@@ -58,7 +58,7 @@ public void filter(ContainerRequestContext request, ContainerResponseContext res
 
         if (setCookie) {
             response.getHeaders().add(HttpHeaderNames.SET_COOKIE,
-                            new NewCookie(Constants.QUERY_COOKIE_NAME, cookieValue, path + id, null, null, NewCookie.DEFAULT_MAX_AGE, false));
+                            new NewCookie(Constants.QUERY_COOKIE_NAME, cookieValue, path + id, null, null, NewCookie.DEFAULT_MAX_AGE, true, true));
```

Comment:
Cookies without HttpOnly=true, secure=true aren't safe.
While the tests pass, I'm not sure how to test this directly.  Perhaps the integration tests will help.